tools/stress_mcp.py

```
@@ -0,0 +1,270 @@
+                        await do_handshake(reader)
+                        await write_frame(writer, json.dumps(apply_payload).encode("utf-8"))
+                        resp = await read_frame(reader)
+                        try:
+                            data = json.loads(resp.decode("utf-8", errors="ignore"))
+                            result = data.get("result", data) if isinstance(data, dict) else {}
+                            ok = bool(result.get("success", False))
+                            if ok:
+                                stats["applies"] = stats.get("applies", 0) + 1
+                            else:
+                                stats["apply_errors"] = stats.get("apply_errors", 0) + 1
+                        except Exception:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)